components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -10,25 +10,24 @@ toc = true
     weight = 10
 +++
 
-## Features in IAM v2
-
-IAM v2 policies allow multiple permissions, separating out policy membership from policy definition for fine-grained control, and includes roles for role-based access control.
-Additionally, IAM v2 allows policy members to be managed directly from the Automate UI.
+IAM v2 policies allow multiple permissions, distinguish policy membership from policy definition for fine-grained control, and include roles for role-based access control.
```

Comment:
Because this is merging into the `force-upgrade` branch--where we are eliminating all references to v1 and v2--I should think we want to delete every such reference to "v2" in the docs as well. Or are we keeping v1 and v2 docs around for some time yet...?

Reply:
There is still a page of IAM v1 docs in force upgrade

Reply:
yes, but that page is about to become legacy documentation.

Reply:
right, but i think it would be good to hold off on this a little bit. i think it will be super confusing to users many of whom may not know they are being force upgraded to iam v2. i agree this needs to be done, but think it might be better to be done when we delete the iam v1 page.

Reply:
@msorens My understanding was that work to remove v1 and v2 references would happen at a later date after the force-upgrade branch was merged. As @susanev mentioned, we already did the work to consolidate v1 documentation and remove it from the rest of the Chef Automate documentation.

Reply:
@msorens My understanding was that work to remove v1 and v2 references would happen at a later date after the force-upgrade branch was merged. As @susanev mentioned, we already did the work to consolidate v1 documentation and remove it from the rest of the Chef Automate documentation.

Reply:
@msorens has made the right call. We help users by clarity. I've made changes here and in #3104